src/components/marker-table/index.css

```diff
@@ -31,3 +31,7 @@
   left: 160px;
   width: 80px;
 }
+
+.treeViewMainColumn.name{
```

Comment:
Could you add a space after `name` here?

Reply:
(it's weird that our css linter doesn't say anything about this though..)

Reply:
Created #1940 for that :)

Reply:
> Could you add a space after `name` here?

Done, @canaltinova.